kura/distrib/RELEASE_NOTES.txt

```diff
@@ -1,3 +1,251 @@
+Eclipse Kura - 3.0.0
+-------------------------------------------------------------------------
+Added Features :
+  * Eclipse Kura Wires
+  * Added OPC UA Eclipse Kura Wires driver
+  * Switched to Equinox 3.11.1
+  * Added support for Java 8 that now is the minimal JVM requirement to run Eclipse Kura
+  * Added support for MQTT over Websockets
+  * Added support for AWS IoT and Azure IoT platforms
+  * Simple JSON is now a supported encoding option in Eclipse Kura Cloud Service
+  * Added support for Fedora 25 systems
```

Comment:
I would limit this to "ARM" support, as standard fedora 25 (x86_64) is not supported.